process_material/models.py

```diff
@@ -0,0 +1,53 @@
+from django.db import models
+from django.contrib.auth.models import (AbstractUser, AbstractBaseUser,
+                                        PermissionsMixin)
+
+class User(AbstractUser):
+
+    archivist_group = [u"rac_archivists"]
+    donor_group = [u"rac_donors"]
+    researcher_group = [u"rac_researchers"]
+
+    email = models.EmailField(
+        verbose_name=_('email address'), max_length=255, unique=True, blank=False)
+    first_name = models.CharField(_('first name'), max_length=150, blank=False)
+    last_name = models.CharField(_('last name'), max_length=150, blank=False)
+    username = models.CharField(_('username'), max_length=150, blank=False)
+
+    is_staff = models.BooleanField(
+        _('staff status'),
+        default=False,
+        help_text=_('Designates whether the user is an RAC staff member.'),
+    )
+
+    def get_full_name(self):
+        """
+        Return the first_name plus the last_name, with a space in between.
+        """
+        full_name = '%s %s' % (self.first_name, self.last_name)
+        return full_name.strip()
```

Comment:
You *could* use the property decorator here:

```suggestion
    @property
    def full_name(self):
        """
        Return the first_name plus the last_name, with a space in between.
        """
        full_name = '%s %s' % (self.first_name, self.last_name)
        return full_name.strip()
```
That way, you could do `User.full_name` rather than `User.get_full_name()` - it's a little cleaner.